js/core/markdown.js

```diff
@@ -1,280 +1,232 @@
-/*global marked*/
-// Module core/markdown
-// Handles the optional markdown processing.
-//
-// Markdown support is optional. It is enabled by setting the `format`
-// property of the configuration object to "markdown."
-//
-// We use marked for parsing Markdown.
-//
-// Note that the content of SECTION elements, and elements with a
-// class name of "note", "issue" or "req" are also parsed.
-//
-// The HTML created by the Markdown parser is turned into a nested
-// structure of SECTION elements, following the strucutre given by
-// the headings. For example, the following markup:
-//
-//     Title
-//     -----
-//
-//     ### Subtitle ###
-//
-//     Here's some text.
-//
-//     ### Another subtitle ###
-//
-//     More text.
-//
-// will be transformed into:
-//
-//     <section>
-//       <h2>Title</h2>
-//       <section>
-//         <h3>Subtitle</h3>
-//         <p>Here's some text.</p>
-//       </section>
-//       <section>
-//         <h3>Another subtitle</h3>
-//         <p>More text.</p>
-//       </section>
-//     </section>
-//
-
-define(
-    ['core/marked'],
-    function (coreMarked) {
-        marked.setOptions({
```

Comment:
So there's something fishy going on here with regards to the global `marked` object and the module `coreMarked` object. My hunch is that I had reasons to do this. I'm not sure why I didn't document them, though. It could also just be a refactoring mistake that I failed to notice because I was then hitting the global marked object.